cli/cluster/errors.go

```diff
@@ -0,0 +1,15 @@
+package cluster
```

Comment:
It doesn't seem to be related to "tt: initial support for cluster config".

Reply:
Sure. Done.